### DISCUSSION TITLE
Clicking "Yes" radio button duplicates Github Sponsors sales tax message Update: My bad, I can't read!

### DISCUSSION THREAD

**Discussion by lewislarsen** [community/community#8197] [Other Feature Feedback, Questions, & Ideas]
**Whoops, I didn't realise the message was different, clearly skimming things is a bad idea! Apologies!**

If you click the Yes radio button for "Are you purchasing as a business?" when it is already selected, it will duplicate the message about sharing certain sponsorship information.

![image](https://user-images.githubusercontent.com/2665668/143777956-e2344948-0688-4c5e-aedb-7f42c6303631.png)

**Comment by jlord** [comment]
Oh, thank you! It is slightly different but I get that it is confusing to be so similar and there twice, I'll let the team know ✨
